components/gfx/text/glyph.rs

```diff
@@ -174,7 +174,7 @@ struct DetailedGlyphRecord {
 
 impl PartialOrd for DetailedGlyphRecord {
     fn partial_cmp(&self, other: &DetailedGlyphRecord) -> Option<Ordering> {
-        self.entry_offset.partial_cmp(&other.entry_offset)
+        Some(self.cmp(other))
```

Comment:
Won't this also compare `details_offset`? The previous code was only comparing `entry_offset`.

Reply:
I think it uses the `Ord` implementation bellow that only compares `entry_offset`, right? Maybe I can move it down so it is clear that it is using `Ord`.